platform.md

```diff
@@ -353,50 +358,69 @@ Usage:
+| `/dev/stderr`                                    | Logs (warnings, errors)                                                                                  |
+| `<group>`                                        | Detected buildpack group  (see [`group.toml`](#grouptoml-toml))                                          |
+| `<plan>`                                         | Resolved Build Plan (see [`plan.toml`](#plantoml-toml))                                                  |
+| `<generated>`                                    | Metadata about generated Dockerfiles from image extensions (see [`generated.toml`](#generatedtoml-toml)) |
+| `<analyzed>`                                     | Updated to include the run image obtained from applying generated Dockerfiles                            |
+| `<output>/<image extension ID>/<run.Dockerfile>` | Generated Dockerfiles (see [Image Extension Specfication](image-extension.md)                            |
+
+| Exit Code       | Result                                                                            |
+|-----------------|-----------------------------------------------------------------------------------|
+| `0`             | Success                                                                           |
+| `11`            | Platform API incompatibility error                                                |
+| `12`            | Buildpack API incompatibility error                                               |
+| `1-10`, `13-19` | Generic lifecycle errors                                                          |
+| `20`            | All buildpacks groups have failed to detect w/o error                             |
+| `21`            | All buildpack groups have failed to detect and at least one buildpack has errored |
+| `22-29`         | Detection-specific lifecycle errors                                               |
+| `80-89`         | Generation-specific lifecycle errors                                              |
 
 The lifecycle:
 - SHALL detect a single group from `<order>` and write it to `<group>` using the [detection process](buildpack.md#phase-1-detection) outlined in the Buildpack Interface Specification
 - SHALL write the resolved build plan from the detected group to `<plan>`
 
+When image extensions are present in the order (**experimental**), the lifecycle:
+- SHALL execute all image extensions in the order defined in `<group>` according to the process outlined in the [Buildpack Interface Specification](buildpack.md).
+- SHALL record all generated run.Dockerfiles in `<generated>`.
+- SHALL copy all generated run.Dockerfiles to `<output>`/generated.
+- SHALL replace the `run-image` reference in `<analyzed>` with the selected run image reference. The selected run image reference SHALL be the base image referenced in the Dockerfile output by the last image extension in the group.
```

Comment:
Another option: instead of updating analyzed.toml, the lifecycle could return the reference in something like generated.toml, and the platform could handle either updating analyzed.toml or providing the new reference to the exporter (this would require bringing back the `-run-image` flag, which was removed in platform 0.7).